services/resource-usage-tracker/tests/unit/with_dbs/conftest.py

```diff
@@ -62,3 +70,75 @@ def mocked_prometheus(mocker: MockerFixture) -> mock.Mock:
         autospec=True,
     )
     return mocked_get_prometheus_api_client
+
+
+@pytest.fixture()
+def user_id() -> UserID:
+    return UserID(randint(1, 10000))
+
+
+@pytest.fixture()
+def user_db(postgres_db: sa.engine.Engine, user_id: UserID) -> Iterator[dict]:
+    with postgres_db.connect() as con:
+        # removes all users before continuing
+        con.execute(users.delete())
+        con.execute(
+            users.insert()
+            .values(
+                id=user_id,
+                name="test user",
+                email="test@user.com",
+                password_hash="testhash",
+                status=UserStatus.ACTIVE,
+                role=UserRole.USER,
+            )
+            .returning(sa.literal_column("*"))
+        )
+        # this is needed to get the primary_gid correctly
+        result = con.execute(sa.select(users).where(users.c.id == user_id))
+        user = result.first()
+        assert user
+        yield dict(user)
+
+        con.execute(users.delete().where(users.c.id == user_id))
+
+
+@pytest.fixture()
+def project_uuid() -> ProjectID:
+    return ProjectID(f"{uuid.uuid4()}")
+
+
+@pytest.fixture()
+def project_db(
+    postgres_db: sa.engine.Engine,
+    project_uuid: ProjectID,
+    user_id: UserID,
+    faker: Faker,
+) -> Iterator[dict]:
```

Comment:
same for projects. no need to duplicate here.

Reply:
i will add it in the upcoming PR today afternoon/tommorow